### PR TITLE
Accordion

### PR DESCRIPTION
**Pull request by HardeepAsrani**
I've finished the Accordion Block. Can you guys test and give feedback?

It adds some new files so you need to run this in the terminal in otter-blocks folder to update the autoloader file:

`composer dumpautoload -o`

Part of https://github.com/Codeinwp/gutenberg-blocks/issues/267

@Soare-Robert-Daniel @mghenciu

### CODE REVIEW

File: src/blocks/accordion/group/save.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+}) => {
+	return (
+		<div
+			id={ attributes.id }
+			className={ classnames(
+				className,
+				{
+					[`is-${ attributes.gap }-gap`]: attributes.gap
+				}
+			) }
```
Comment (Soare-Robert-Daniel) [inline src/blocks/accordion/group/save.js:23]: `classNames` is undefined

File: src/blocks/accordion/item/save.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ */
+const { InnerBlocks, RichText } = wp.blockEditor;
+
+const Save = ({
+	attributes,
+	className
+}) => {
+	return (
+		<details
+			className={ className }
```
Comment (Soare-Robert-Daniel) [inline src/blocks/accordion/item/save.js:12]: `className` is undefined

**Review by Soare-Robert-Daniel — CHANGES_REQUESTED** [review]
This is how it looks in Vivaldi which is a Chrome-based browser, like Edge.

image

File: src/blocks/accordion/group/save.js (hunk 1 above)
Comment (HardeepAsrani) [reply src/blocks/accordion/group/save.js:23]: I've just kept it like this as it's the same for the rest of the blocks. When we merge TS, we can change this.

**Comment by HardeepAsrani** [comment]
> This is how it looks in Vivaldi which is a Chrome-based browser, like Edge.
> 
> image

Should be fixed. @Soare-Robert-Daniel

**Review by Soare-Robert-Daniel — APPROVED** [review]

**Comment by Soare-Robert-Daniel** [comment]
All good. No problems so far.

**Pull request merged**